README.md

```diff
@@ -15,31 +15,42 @@ an AWS account.
 
 ### Install
 
-```bash
-mkvirtualenv repokid
-git clone git@github.com:Netflix/repokid.git
-cd repokid
-python setup.py develop
-repokid config config.json
-```
+There are multiple methods of install for `Repokid`:
+
+For users intending to only operate and not develop:
+
+        pip install repokid
+
+Otherwise, contributors/developers can use:
+
+        mkvirtualenv repokid
```

Comment:
Does `mkvirtualenv` automatically activate the newly created env?

Reply:
I'm actually unsure, I haven't tested this part because I don't use `mkvirtualenv`.